CMakeLists.txt

```diff
@@ -119,6 +119,8 @@ add_library(global-libs INTERFACE)
 
 include (cmake/sanitize.cmake)
 
+include (cmake/instrument.cmake)
```

Comment:
`intrument.cmake` as a name is too generic, IMHO. Maybe `xray_instrumentation.cmake`?